changelogs/current.yaml

```diff
@@ -112,6 +112,11 @@ minor_behavior_changes:
 - area: tracers
   change: |
     Set status code based on GRPC status code for OpenTelemetry tracers (previously unset).
+- area: xds-failover
```

Comment:
Hmmm... I can see your point that this may not be needed...
On the other hand, this PR adds a runtime flag, and feels wrong not to document it.
I've tried to clarify the release note.
I'll leave the decision whether to keep this or not to the senior-maintainer.

Reply:
ah, my suggestion was to keep release notes but to mention that this is a test feature.